pkg/registry/resourcequota/strategy.go

```diff
@@ -106,9 +106,8 @@ func MatchResourceQuota(label labels.Selector, field fields.Selector) generic.Ma
 }
 
 // ResourceQuotaToSelectableFields returns a label set that represents the object
-// TODO: fields are not labels, and the validation rules for them do not apply.
 func ResourceQuotaToSelectableFields(resourcequota *api.ResourceQuota) labels.Set {
 	return labels.Set{
-		"name": resourcequota.Name,
+		"metadata.name": resourcequota.Name,
```

Comment:
Discussed this with @bgrant0607 and sent https://github.com/kubernetes/kubernetes/pull/13457 to delete that